server/graph/schema.graphqls

```diff
@@ -418,15 +418,16 @@ input UpdateUserInput {
 }
 
 input ForgotPasswordInput {
-  email: String!
+  email_or_phone: String!
```

Comment:
Its better to keep `email` & add `phone_number`. Make both the fields optional and add validation in API that either of them should be present.

This will require less updates on SDK end and will also keep our APIs consistent